src/infra/services/service.ts

```
@@ -0,0 +1,47 @@
+import { agent } from "@pagopa/ts-commons";
+import {
+  AbortableFetch,
+  setFetchTimeout,
+  toFetch,
+} from "@pagopa/ts-commons/lib/fetch";
+
+import * as E from "fp-ts/lib/Either";
+
+import { UrlFromString } from "@pagopa/ts-commons/lib/url";
+import { pipe } from "fp-ts/lib/function";
+import { Millisecond } from "@pagopa/ts-commons/lib/units";
+import { config } from "../../app/config";
+
+export const basePath = pipe(
+  config,
+  E.chain((config) =>
+    pipe(
+      config.service.serviceBasePath,
+      UrlFromString.decode,
+      E.mapLeft(() => new Error("Invalid service parameters"))
```

don't swallow decoding errors